types/file.ts

```diff
@@ -3,7 +3,7 @@ import { Buffer } from 'buffer';
 /**
  * Data for files to upload
```

Comment:
"to upload" or "currently uploading"? The description tells one, but the name tells another.